README.md

```diff
@@ -3,6 +3,14 @@
 This repository contains code for the Elastic Open Web Crawler.
 This is a tool to allow users to easily ingest content into Elasticsearch from the web.
 
+⚠️ _The Open Crawler is currently in **tech-preview**_.
+Tech-preview features are subject to change and are not covered by the support SLA of generally available (GA) features.
+Elastic plans to promote this feature to GA in a future release.
+
+ℹ️ The Open Crawler requires a running instance of Elasticsearch to index documents into.
+If you don't have this set up yet, check out the [quickstart guide for Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/quickstart.html) to get started.
+_Open Crawler `v0.1` is confirmed to be compatible with Elasticsearch `v8.13.0` and above._
+
 ## How it works
```

Comment:
`Side note`

I would rework this into a super high-level summary of the user actions required to get setup and launch crawl. Something like this:

- Have a running Elasticsearch instance
- Run crawler service
- Configure yaml file
- Launch crawl


Also _The crawl results can be output in 3 different modes:_ kind of confuses me and raises the question "Do I actually need to have Elasticsearch running?"

Reply:
> "Do I actually need to have Elasticsearch running?"

You don't technically. You can run crawler to output to file and it will never interact with Elasticsearch.

Reply:
> I would rework this into a super high-level summary of the user actions required to get setup and launch crawl

I feel like this is covered in "Setup". I've addressed the other comments, do you still think this section should be rearranged with the latest changes?

Reply:
For me "How it works" should summarize all the actions I as a user have to take to get this working. Otherwise, I have to infer the overall workflow by scanning the subheadings.

I don't think the current _how it works_ info is complete and most of the info is a list of the output options.

> Crawler runs crawl jobs on command based on config files in the config directory. 1 URL endpoint on a site will correlate with 1 result output.

I'd prefer to know how the whole process works first, or at least make this more active and direct from the user perspective

Reply:
> "Do I actually need to have Elasticsearch running?"
You don't technically. You can run crawler to output to file and it will never interact with Elasticsearch.

Right but I think the default path for us here should be to write to Elasticsearch and mention elsewhere that you can output to file or print to terminal.

Otherwise we're putting the "connecting to Elasticsearch" instructions last despite these being a prerequisite for the default path.

Reply:
Fair enough, I've expanded that section and removed mention of other output types (users can figure that out from reading the configuration file documentation anyway)